src/porepy/geometry/point_in_polyhedron_test.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import annotations
+
+from typing import Any
+
+import numpy as np
+
+
+class PointInPolyhedronTest(object):
```

Comment:
```suggestion
class PointInPolyhedronTest:
```
No need to inherit from object in python 3.